visualization/app/codeCharta/ui/ribbonBar/colorMetricChooser/colorMetricChooser.component.spec.ts

```diff
@@ -24,14 +25,24 @@ describe("colorMetricChooserComponent", () => {
 		Store.dispatch(setColorMetric("aMetric"))
 		await render(ColorMetricChooserComponent, { excludeComponentDeclaration: true })
 
+		expect(screen.getByRole("combobox").getAttribute("aria-disabled")).toBe("false")
+
 		await userEvent.click(await screen.findByText("aMetric (1)"))
 		expect(screen.getByText("Color Metric (highest value)")).not.toBe(null)
 		const options = screen.queryAllByRole("option")
-		expect(options[0].textContent).toMatch("aMetric (1)")
-		expect(options[1].textContent).toMatch("bMetric (2)")
+		await expect(options[0].textContent).toMatch("aMetric (1)")
```

Comment:
this `await` is not necessary, as [Jest's `toMatch`](https://jestjs.io/docs/26.x/expect#tomatchregexp--string) expectation returns not a Promise. 

The IDE gets confused, as `expect` is set globally and therefore it doesn't know if the `expect` comes from jest (our unit tests) or jest/expect-puppeteer (our e2e tests). 
![image](https://user-images.githubusercontent.com/3596742/193408845-a5f0e2e9-00f2-4e13-a3f8-7a154a980e4f.png)

Do you mind to create a new bug ticket for this falsely IDE hints?

Reply:
Thanks for finding it out! I will change it and Luis opens a new ticket.